includes/overlords/contribs.php

```diff
@@ -570,4 +570,162 @@ public static function build_sort()
 
 		return $sort;
 	}
+
+	/**
+	 * Create a feed either for an individual contribution or for all contributions
+	 * @param $template
+	 * @param $helper
+	 * @param $path_helper
+	 * @param mixed $contrib
+	 * @return \Symfony\Component\HttpFoundation\Response
+	 * @throws Exception
+	 */
+	public static function build_feed($template, $helper, $path_helper, $contrib = false)
+	{
+		$contrib_id = ($contrib) ? $contrib->contrib_id : false;
+
+		if (!phpbb::$config['feed_overall'])
+		{
+			// Don't proceed if feeds are disabled
+			trigger_error('NO_FEED_ENABLED');
+		}
+
+		// Show one contribution only, if on the specific contrib page
+		$contrib_specific = ($contrib_id) ? 'AND r.contrib_id = ' . (int) $contrib_id : '';
+
+		$sql_ary = [
+			'SELECT' => 'r.*, c.*, u.username_clean',
+
+			'FROM' => [
+				TITANIA_REVISIONS_TABLE => 'r',
+				TITANIA_CONTRIBS_TABLE => 'c',
+				USERS_TABLE => 'u',
+			],
+
+			'WHERE'	=> 'r.revision_status = ' . ext::TITANIA_REVISION_APPROVED . '
+				AND r.revision_submitted = 1
+				AND c.contrib_status = ' . ext::TITANIA_CONTRIB_APPROVED . '
+				AND r.contrib_id = c.contrib_id
+				AND u.user_id = c.contrib_user_id 
+				' . $contrib_specific,
+
+			'ORDER_BY'	=> 'r.validation_date DESC',
+		];
+
+		$sql = phpbb::$db->sql_build_query('SELECT', $sql_ary);
+		$result = phpbb::$db->sql_query_limit($sql, 100);
+
+		$rows = [];
+		$feed_updated_time = false;
+
+		while ($row = phpbb::$db->sql_fetchrow($result))
+		{
+			// Only proceed if the contribution is not in categories that are all hidden
+			if (!self::feed_hidden_category_check($row['contrib_id']))
+			{
+				$feed_rows = [];
+				$feed_rows['item_date'] = date(\DateTime::ATOM, $row['validation_date']);
+
+				// Get the most recent time
+				if (!$feed_updated_time)
+				{
+					$feed_updated_time = $row['validation_date'];
+				}
+
+				// Make the name including the version
+				$feed_rows['item_title'] = $row['contrib_name'] . ' ' . $row['revision_version'];
+
+				if ($row['revision_name'])
+				{
+					// Include the code name if it's supplied
+					$feed_rows['item_title'] .= ' (' . $row['revision_name'] . ')';
+				}
+
+				$feed_rows['item_author'] = $row['username_clean'];
```

Comment:
I left a comment here: https://github.com/phpbb/customisation-db/commit/ea1d151fc755175e78914de30fc8a3160ada846e#r35146186

`$row['username']` should be used instead.